package.json

```diff
@@ -145,9 +145,9 @@
   "scripts": {
     "start": "node ./dist/index.js",
     "doc:gen": "jsdoc --configure .jsdoc.json --verbose",
-    "predocker:dev": "npm install && npm run db:setup && babel-node ./scripts/db_restore.js opencollective_dvl",
+    "predocker:dev": "npm install && npm run db:setup && babel-node --presets es2015,stage-0 --plugins add-module-exports,lodash ./scripts/db_restore.js opencollective_dvl",
```

Comment:
Is there a way to refactor this `babel-node . --presets es2015,stage-0 --plugins add-module-exports,lodash` so that we just have these parameters in one place?

Reply:
I'll double check and see if that's possible

Reply:
Good to merge this then?

Reply:
@HipsterBrown Have you created the issue?

Reply:
I'm working on the update today